README.adoc

```diff
@@ -472,6 +472,55 @@ spec:
 ...
 ----
 
+== Experimental
+
+=== Observing a Remote Cluster
+
+There are certain use cases where Kiali needs to be deployed in one cluster (Control Plane) and observe a different cluster (Data Plane). link:https://user-images.githubusercontent.com/6889074/87819080-ad099980-c839-11ea-834b-56eec038ce4d.png:[Diagram].
+
+Follow these steps:
+
+1: You should have one of the central istiod setups running
+
+  - https://github.com/istio/istio/wiki/Central-Istiod-manages-remote-data-plane
+
+  - https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps
+
+2: Deploy the link:https://github.com/istio/istio/blob/master/samples/addons/kiali.yaml[Kiali ClusterRole, ClusterRoleBinding, and ServiceAccount] to the Data Plane
```

Comment:
would not this install kiali as well?

Reply:
also do you mean data plane cluster here?

Reply:
So this step is just to create the Kiali specific ClusterRole, ClusterRoleBinding, and ServiceAccount to the Data Plane. Not the Kiali Deployment, Service, etc.

Reply:
i see.   basically just portions of the kiali yaml.  You can check if Kiali team allows you to configure it via their operator.

Reply:
I took a quick look at the [spec](https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml) and it doesn't seem like it allows you to turn entire resources on/off like Istio.
We'll need to add 2 options in the future
1. Only include the CR, CRB, and SA
2. Exclude the CR, CRB, and SA and add volumes to the Deployment

Lots to learn :)